docs/content/latest/api/ycql/expr_fcall.md

```diff
@@ -60,6 +61,28 @@ function_call ::= function_name '(' [ arguments ... ] ')'
 | MIN | Returns the minimum value of column values |
 | MAX | Returns the maximum value of column values |
 
+
+## Partition_hash function
+`partition_hash` is a function that takes as arguments the values of partition keys of a row and returns an `uint16` hash value.
+The full hash partition value spans from `0 - 65535`. Each tablet keeps are range of values and the hash is used to decode on which tablet the row will reside.  
+We can also use `partition_hash` as a filter in the `WHERE` clause to query tablets directly based on their hash-range. 
```

Comment:
We can start this para with:

`partition_hash` can be handy for getting querying a subset of the data to get approximate row counts or to breakdown full-table operations into smaller sub-tasks that can be parallized.


### Querying a subset of the data
One use of `partition_hash` can be used to query a subset of the data and get approximate count of rows in the table.

<describe the example here>

### Parallel full table scans

<Describe some details here.>